src/main/java/com/dev/cinema/controllers/InjectDataController.java

```diff
@@ -0,0 +1,42 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.model.Role;
+import com.dev.cinema.model.User;
+import com.dev.cinema.service.RoleService;
+import com.dev.cinema.service.ShoppingCartService;
+import com.dev.cinema.service.UserService;
+import java.util.List;
+import javax.annotation.PostConstruct;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class InjectDataController {
```

Comment:
better rename class to DataInitializer and use Component annotation instead of the Controller

Reply:
renamed